src/data-workspace/data-entry-cell/data-entry-cell.module.css

```diff
@@ -12,41 +12,35 @@
 .cellInnerWrapper {
     width: 100%;
     height: 100%;
+    /* Make this a coordinate context for corner indicators */
     position: relative;
 }
-
-.input {
-    width: 100%;
-    height: 100%;
-    background: none;
-    border: none;
-    padding: 8px 16px 8px 8px;
-}
-.input:disabled {
-    background: var(--colors-grey300);
-}
-.input:hover:not(:disabled) {
+.cellInnerWrapper:hover:not(.disabled) {
```

Comment:
Styles in here are refactored to work for different input types that might be rendered